### ISSUE TITLE
When I use @font-face its formating all of the nesting element to the global scope on save like it doesn't recognize sass type..

### ISSUE THREAD

**Issue by omersname** [TheRealSyler/sass-formatter#6]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
If you add screen shots of code, please also paste the code as text so that i can just copy the code and reproduce the bug.

**Desktop (please complete the following information):**
 - OS: [e.g. windows 10]
 - Version (of the extension) [e.g. 1.6]

**Additional context**

**Comment by TheRealSyler** [comment]
please elaborate on the issue that you are having.

**Comment by omersname** [comment]
If I write
```
@font-face
      font-family: 'font'
      src: url(../fonts/font.type)
```
when I save its get formated like:
```
@font-face
font-family: 'font'
src: url(../fonts/font.type)
```
and then my compiler doesnt recognize it because it is nesting sensitive..

**Comment by TheRealSyler** [comment]
fixed in b4d2aaf8034009ea35b6740230e78141ce542197
